src/shared/ui/video-playlist-with-player/playlist/playlist.module.scss

```
@@ -0,0 +1,95 @@
+.playlist {
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  gap: 8px;
+}
+
+.videos-container {
+  scrollbar-width: thin;
+
+  overflow-y: scroll;
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+
+  @include media-tablet {
+    overflow: scroll auto;
+    flex-direction: row;
+    gap: 24px;
+  }
+}
+
+.playlist-title {
+  margin: 0;
+  font-size: 24px;
+  line-height: 32px;
+}
+
+.playlist-item {
+  cursor: pointer;
+
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+
+  margin-right: 5px;
+  padding: 16px;
+
+  &:hover {
+    background-color: $color-gray-100;
+
+    @include media-tablet {
+      background-color: initial;
+    }
+  }
+
+  p {
+    margin: 0;
+    line-height: 24px;
+  }
+
+  &.selected {
+    p {
+      font-weight: $font-weight-bold;
+    }
+  }
+
+  &-thumbnail-container {
+    position: relative;
+    display: flex;
+    justify-content: center;
+  }
+
+  img {
+    display: block;
+    max-width: 100%;
+    border-radius: 8px;
+
+    @include media-tablet {
+      max-width: 200px;
+    }
+  }
+
+  @include media-tablet {
+    margin-bottom: 10px;
+    padding: 0;
+  }
+}
+
+.now-playing-overlay {
+  position: absolute;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  width: 100%;
+  height: 100%;
+
```

design inconsistency

```suggestion
  font-size: 14px;
```

---

fixed